matrix-sdk-android/src/main/java/im/vector/matrix/android/internal/session/notification/DefaultPushRuleService.kt

```diff
@@ -131,10 +131,20 @@ internal class DefaultPushRuleService @Inject constructor(private val getPushRul
         }
     }
 
-    fun dispatchRoomLeft(roomid: String) {
+    fun dispatchRoomJoined(roomId: String) {
         try {
             listeners.forEach {
-                it.onRoomLeft(roomid)
+                it.onRoomJoined(roomId)
+            }
+        } catch (e: Throwable) {
+            Timber.e(e, "Error while dispatching room left")
+        }
+    }
+
+    fun dispatchRoomLeft(roomId: String) {
+        try {
+            listeners.forEach {
+                it.onRoomLeft(roomId)
```

Comment:
should catch inside each iteration and continue?

Reply:
done